packages/nvidia_nat_strands/tests/test_strands_llm.py

```
@@ -82,15 +76,28 @@ async def test_api_type_validation(self, mock_model, openai_config_wrong_api, mo
                 pass
         mock_model.assert_not_called()
 
+    @pytest.mark.parametrize("verify_ssl", [True, False], ids=["verify_ssl_true", "verify_ssl_false"])
+    @patch("openai.AsyncOpenAI")
+    @patch("strands.models.openai.OpenAIModel")
+    @pytest.mark.asyncio
+    async def test_verify_ssl_passed_to_client(self,
+                                               mock_model,
+                                               mock_async_openai,
```

Ruff (0.15.5)

[warning] Unused method argument: `mock_async_openai`

(ARG002)